### PR TITLE
Move more samples/tests to new logger

### PR DESCRIPTION
**Pull request by nashif**
Move remaining samples/tests to new logger. We still have bluetooth and a few
net samples that need to be moved.

### CODE REVIEW

**Comment by jukkar** [comment]
> a few net samples that need to be moved

I tried earlier to convert all the networking code to use new logger, can you point me to those samples that still need conversion?

**Comment by nashif** [comment]
@jukkar 

```
net/coap_client/prj_bt.conf:CONFIG_SYS_LOG=y
net/coap_client/prj_bt.conf:CONFIG_SYS_LOG_SHOW_COLOR=y
net/echo_client/boards/nrf52840_pca10056.conf:CONFIG_SYS_LOG_SHOW_COLOR=y
net/echo_server/boards/nrf52840_pca10056.conf:CONFIG_SYS_LOG_SHOW_COLOR=y
net/echo_server/overlay-cc2520.conf:CONFIG_SYS_LOG_IEEE802154_DRIVER_LEVEL=1
net/lwm2m_client/overlay-wncm14a2a.conf:CONFIG_SYS_LOG_MODEM_LEVEL=4
net/rpl-mesh-qemu/node/prj_qemu.conf:CONFIG_SYS_LOG=y
net/rpl-mesh-qemu/node/prj_qemu.conf:CONFIG_SYS_LOG_NET_LEVEL=4
net/rpl-mesh-qemu/node/prj_qemu.conf:CONFIG_SYS_LOG_SHOW_COLOR=y
net/rpl-mesh-qemu/node/prj_qemu.conf:CONFIG_SYS_LOG_IEEE802154_DRIVER_LEVEL=2
net/rpl-mesh-qemu/root/prj_qemu.conf:CONFIG_SYS_LOG=y
net/rpl-mesh-qemu/root/prj_qemu.conf:CONFIG_SYS_LOG_NET_LEVEL=4
net/rpl-mesh-qemu/root/prj_qemu.conf:CONFIG_SYS_LOG_SHOW_COLOR=y
net/rpl-mesh-qemu/root/prj_qemu.conf:CONFIG_SYS_LOG_IEEE802154_DRIVER_LEVEL=2
net/sockets/echo/boards/cc3220sf_launchxl.conf:#CONFIG_SYS_LOG=y
net/ws_console/prj_tls.conf:CONFIG_SYS_LOG_NET_LEVEL=2
net/ws_console/prj_tls.conf:CONFIG_SYS_LOG_SHOW_COLOR=y
net/ws_echo_server/prj_tls.conf:CONFIG_SYS_LOG_NET_LEVEL=2
net/ws_echo_server/prj_tls.conf:CONFIG_SYS_LOG_SHOW_COLOR=y
```


and
```

net/app/prj-no-ipv4.conf:CONFIG_SYS_LOG_SHOW_COLOR=y
net/app/prj-no-ipv4.conf:#CONFIG_SYS_LOG_NET_LEVEL=4
net/app/prj-no-ipv6.conf:CONFIG_SYS_LOG_SHOW_COLOR=y
net/app/prj-no-ipv6.conf:#CONFIG_SYS_LOG_NET_LEVEL=4
net/app/prj-with-dns.conf:CONFIG_SYS_LOG_SHOW_COLOR=y
net/app/prj-with-dns.conf:#CONFIG_SYS_LOG_NET_LEVEL=4
net/ieee802154/l2/src/ieee802154_fake_driver.c:#define NET_SYS_LOG_LEVEL 4
net/ieee802154/l2/src/ieee802154_test.c:#define NET_SYS_LOG_LEVEL 4
```

**Comment by jukkar** [comment]
Ok, those were not part of sanity checked runs then. I will probably remove most of them, and convert + add to sanity runs those that make sense.

**Review by dcpleung — APPROVED** [review]

File: samples/subsys/usb/webusb/src/webusb_serial.c [hunk 1]
```diff
@@ -287,35 +288,35 @@ static void webusb_serial_dev_status_cb(enum usb_dc_status_code status,
 {
 	ARG_UNUSED(param);
 
-	SYS_LOG_DBG("");
+	LOG_DBG("");
```
Comment (finikorg) [inline samples/subsys/usb/webusb/src/webusb_serial.c]: I do not think those debugs are useful without function names, probably we should use USB_DBG()

**Pull request merged**